.github/workflows/major-release.yml

```diff
@@ -0,0 +1,47 @@
+name: Major Release
+
+on:
+  schedule:
+    - cron: 0 0 15 2,8 *  # runs at midnight UTC every 15 February and 15 August
+
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      contents: read
```

Comment:
I think we want to keep `contents: read` on the top level, so if we add more jobs in follow-up PRs, they inherit the stricter permission

Reply:
If we don't define `permission.contents` globally, it will use `read`/`none` by default. So I think that's the case now, right?

Reply:
Explicit is better than implicit, I would still do the change